src/export/digitalPlanning/mocks/notificationOfCommencement.ts

```diff
@@ -1,72 +1,29 @@
-export const mockNOCSession = {
-  id: "56750f3b-e94f-4163-817d-b7baa10cac67",
+import { Session } from "../../../types/session.js";
+
+// Not fee-carrying = no `data.govUkPayment`
+export const mockNOCSession: Session = {
```

Comment:
Important to note that all mocked sessions now adhere to `Session` type! Previously they matched the output of the `/summary` admin endpoint.